sdks/java/io/kafka/pom.xml

```diff
@@ -106,7 +106,7 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka-clients</artifactId>
-      <version>0.9.0.1</version>
+      <version>[0.9.0.1,0.11.0)</version>
```

Comment:
I'm sorry for the confusion here; we should revert back to the previous variant _without_ a version range here. Ranges are not intended to communicate version compatibility, and not recommended by Maven itself in a scenario like this one. This has to be a specific version number; we can add to the comment above the range that is expected to work.

(Sorry @rangadi for stepping on your toes here.)